bedrock/products/tests/test_helper_misc.py

```diff
@@ -119,301 +142,529 @@ def render(s, context=None):
 class TestVPNSubscribeLink(TestCase):
     rf = RequestFactory()
 
-    def _render(self, entrypoint, link_text, plan="12-month", class_name=None, lang=None, optional_parameters=None, optional_attributes=None):
+    def _render(
+        self,
+        entrypoint="www.mozilla.org-vpn-product-page",
+        link_text="Get Mozilla VPN",
+        plan="12-month",
+        class_name="mzp-c-button",
+        country_code=None,
+        lang=None,
+        optional_parameters=None,
+        optional_attributes=None,
+    ):
         req = self.rf.get("/")
         req.locale = "en-US"
         return render(
-            "{{{{ vpn_subscribe_link('{0}', '{1}', '{2}', '{3}', '{4}', {5}, {6}) }}}}".format(
-                entrypoint, link_text, plan, class_name, lang, optional_parameters, optional_attributes
+            "{{{{ vpn_subscribe_link('{0}', '{1}', '{2}', '{3}', '{4}', '{5}', {6}, {7}) }}}}".format(
+                entrypoint, link_text, plan, class_name, country_code, lang, optional_parameters, optional_attributes
```

Comment:
In the future, it might be nice to use f-strings to make these easier to scan:

```
f"{{{{ vpn_subscribe_link('{entrypoint}', '{link_text}', '{plan}', '{class_name}', '{country_code}', '{lang}', {optional_parameters}, {optional_attributes}) }}}}"
```

Reply:
Ah, yeah that would be much nicer 👍